app/helpers/games_helper.rb

```diff
@@ -1,2 +1,11 @@
 module GamesHelper
+  
+  def cell_black(row, column)
+    if row % 2 == 0 && column % 2 == 0 || row % 2 != 0 && column % 2 != 0
```

Comment:
Last one @nerdycat99 and then just go ahead and merge it!

The whole method can be just one line:
```
def ...
row % 2 == 0 && column % 2 == 0 || row % 2 != 0 && column % 2 != 0
end
```

since `if` already returns true or false thats all we need to return from the method 😄